tests/slice_selector.json

```diff
@@ -610,6 +610,28 @@
         9
       ]
     },
+    {
```

Comment:
Would it make sense to add similar checks for the "start" and "step" parameters?

Reply:
Agreed.  Let's go ahead and add those tests, then this should be good.

Reply:
I have duplicated them now for start and step, please let me know if it is ok like this.